academy/3-cosmos-hub/3-interchain-security

```diff
@@ -0,0 +1,77 @@
+
+<HighlightBox type="docs">
+
+If you'd like to learn more about how these tasks are executed, you can find a detailed explanation in the [specification](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#channel-initialization).
+
+</HighlightBox>
+
+## Why Use Interchain Security?
+
+There are many benefits for application developers to consume their security through the Cosmos Hub. Besides the obvious security bootstrapping that it enables, chains obtain a strong brand association with the Hub as well.
+
+However, before considering using Interchain Security for your dApp, it remains important to keep in mind why Cross Chain Validation was brought to the Cosmos Hub in the first place. Interchain Security was designed to enable _Hub minimalism_, which is a philosophy that positions the Cosmos Hub as a relatively light-weight and simple service provider to the Interchain. For a Hub to be secure, the number of attack surfaces need to be minimized as much as possible. So, instead of launching a plethora of features directly on the Hub, additional functionality can be extended onto sovereign chains, thus improving the Cosmos network's resilience. If one specific service goes offline, others can continue to function.
+
+This was an argument that was made apparent when [a governance proposal](https://www.mintscan.io/cosmos/proposals/69) recommended that the smart contracting language [CosmWasm](../1-what-is-cosmos/2-cosmos-ecosystem.md#cosmwasm-multi-chain-smart-contracts) would be added to the Cosmos Hub. The community widely believed that a smart contracting platform would be useful to the Hub, but that it would not necessarily be required to run directly on the Hub itself, and that it potentially could have negative implications for the security of the chain.
+
+As such, one of the primary questions app developers need to answer when considering Interchain Security is whether or not your application is aligned with the Cosmos Hub's mission.
+
+## Building With Interchain Security
+
+Interchain Security significantly changes the way you design the tokenomics of your chain, which could potentially impact the design of your app itself. Before building out your chain to run on Interchain Security, it is recommend to make your intentions clear through a post on the [Cosmos Hub forum](https://forum.cosmos.network/) and potentially in the future, a governance proposal. 
```

Comment:
> Interchain Security significantly changes the way you design the tokenomics of your chain

This is fair, but is not so clear cut. All it ICS does is make it so that a consumer chain's tokens cannot be staked on PoS validators. They can still be staked on "Representatives" who perform all of the roles of a validator in governance, but don't run a node.

Reply:
Might be easiest to just remove the sentence

> Interchain Security significantly changes the way you design the tokenomics of your chain, which could potentially impact the design of your app itself. 

And leave it at that